### PR TITLE
Combased-wl

### PR DESCRIPTION
**Pull request by combased-dev**
Creating WL with the logic, as it was discussed before :)

### CODE REVIEW

**Review by sasurobert — DISMISSED** [review]

File: LKMEX-whitelist/combased-custom [hunk 1]
```diff
@@ -0,0 +1,29 @@
+## Name of the project
+Comverse
+
+## Description of the project
+Combased project is web3 agency, a leader in e-commerce space & core DeFi protocol creator. Read more here https://combased.io
+
+## Purpose
+We send to our holders monthly rewards with LKME.
```
Comment (iulianpascalau) [inline LKMEX-whitelist/combased-custom]: LKMEX instead of LKME?

File: LKMEX-whitelist/combased-custom [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+To farm XMEX on the dex we use our CDO wallet, which takes 50% of all royalties and earnings from our services. We provide web3 development services, JewelSwap and sell our Shopify apps.
+
+To qualify all eligible addresses we use our first open source script, which was actually instrumental in developing Elrond ecosystem.
```
Comment (iulianpascalau) [inline LKMEX-whitelist/combased-custom]: No Elrond. We are now MultiversX.

File: LKMEX-whitelist/combased-custom [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+https://github.com/Combased/elrond-lkmex-distribution
```
Comment (iulianpascalau) [inline LKMEX-whitelist/combased-custom]: maybe change the repo, also? Remove elrond?

**Review by iulianpascalau — CHANGES_REQUESTED** [review]

File: LKMEX-whitelist/combased-custom (hunk 3 above)
Comment (combased-dev) [reply LKMEX-whitelist/combased-custom]: Updated, yes, removed :)

**Review by iulianpascalau — APPROVED** [review]

**Review by sasurobert — APPROVED** [review]

**Comment by PABourdais** [comment]
@iulianpascalau can you explain me how in this case you are sure that XMEX are not sent to wallets that don't hold the NFT from the project ?

**Comment by combased-dev** [comment]
> @iulianpascalau can you explain me how in this case you are sure that XMEX are not sent to wallets that don't hold the NFT from the project ?

@PABourdais we have a script, which extracts only holders and adds them. If we make mistake, it is our issue :)

**Comment by PABourdais** [comment]
> > @iulianpascalau can you explain me how in this case you are sure that XMEX are not sent to wallets that don't hold the NFT from the project ?
> 
> @PABourdais we have a script, which extracts only holders and adds them. If we make mistake, it is our issue :)

I was asking to @iulianpascalau if he can explain me how MultiversX team is sure that you are distributing XMEX to holders of Combased NFT, how they can be sure of that ? 

The last time I proposed a SC here, it was a SC that distribute XMEX to a list of whitelisted wallets which they refused.
@iulianpascalau, What's the difference between sending a token and claiming a token ? 

The answer from MultiversX team was:
> You need to create contracts where it is clear that it is used for NFT staking. The contract is too general and can be used for wrong secondary markets.

@iulianpascalau, Isn't this SC too general ? It means that we can do a staking SC that can distribute XMEX to wallets without having to be a holder, we can use it too build a secondary market !

I know that you guys at Combased are good guys and will not cheat on this, but I am shocked to see that they are approving this SC

**Comment by PABourdais** [comment]
So it's merged now @iulianpascalau, I would love to know how you will be sure that this SC is used for this specific Combeys NFTs collection

If there is no answer, I will assume that you know that you did accept a SC without being able to know that it will be used for the good purpose, and that everything relies on the good intention of Combeys project owner.

**Comment by sasurobert** [comment]
Why are you attacking again? Combased have revised and rewritten the SC according to specs. The things were clear. SC which are too general will not be whitelisted, only those with clearly defined structure and who and why gets wrapped xMEX. Your contract was whitelisted too

**Comment by PABourdais** [comment]
I have no more trust in MultiversX team

**Pull request merged**